HISTORY.md

```diff
@@ -3,6 +3,7 @@
 ### New Features
 
 ### Public API Change
+* Transaction::GetForUpdate is extended with a do_validate parameter with default value of true. If false it skips validating the snapshot before doing the read. Similarly ::Merge, ::Put, ::Delete, and ::SingleDelete are extended with assume_tracked with default value of false. If true it indicates that call is assumed to be after a ::GetForUpdate(do_validate=false).
```

Comment:
I think it's more accurate to say that assume_tracked means the call is assumed to be called after _any_ GetForUpdate call right?